OWNERS_ALIASES

```diff
@@ -0,0 +1,22 @@
+# See the OWNERS docs: https://git.k8s.io/community/docs/devel/owners.md
+
+aliases:
+  sig-cluster-lifecycle-leads:
+    - lukemarsden
+    - luxas
+    - roberthbailey
+    - timothysc
+  cluster-api-admins:
```

Comment:
Does this come with github repo privileges like being able to create labels and other things? Is this meant to mirror that structure? If so, then this list should mirror the github organizational structure. It's fine to discuss who should have that level of access here though and then mirror it back to github.

Reply:
In the kube-deploy repo, this list came from the github group with the same name. If we aren't going to be setting up groups in this org then there isn't any point separating out admins from maintainers here.